File: 412-fizzbuzz/fizzbuzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
 var fizzBuzz = function(n) {
    answer = []
   for (let i=1;i<n+1;i++){
          console.log(i)
           if ((i % 5 == 0) && (i % 3 == 0)){
               answer.push('FizzBuzz')}
         
           else if (i % 5==0){
               answer.push('Buzz')}
      
           else if (i % 3==0){
               answer.push('Fizz')}
           else {answer.push(i.toString())}
               
   }
   return answer
};

File: 412-fizzbuzz/fizzbuzz.rb
# @param {Integer} n
# @return {String[]}
def fizz_buzz(n)
    return (1..n).map{|i| 
        next 'FizzBuzz' if 0 == i%15
        next 'Buzz' if (i%5).zero?
        next 'Fizz' if (i%3).zero?
        i.to_s
    }
end
